config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-cluster-api/capk-manager:dev
+      - image: registry:5000/capk-manager-amd64:dev
```

Comment:
This change is likely problematic. I think we'll need a different kustomize layer for kubevirtci

Reply:
@rmohr do you think the right path forward here is to create a unique `/config/kubevirtci/` ?

Reply:
Yes sounds great : +1. We can the drop the 

```
make set-manifest-image
```

in the kubevirtci script.

Reply:
fixed